pkg/cli/init.go

```diff
@@ -50,9 +58,17 @@ Upstream can be one of:
 	cmd.Flags().Bool("rm-asset-dest", false, "Always remove asset destinations if already present")
 	cmd.Flags().Int("retries", 3, "Number of times to retry retrieving upstream")
 
-	viper.BindPFlags(cmd.Flags())
-	viper.BindPFlags(cmd.PersistentFlags())
-	viper.AutomaticEnv()
-	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	// optional flags for "ship init replicated.app"
+	cmd.Flags().String("customer-id", "", "Customer ID for which to query app specs. Required for all ship operations.")
```

Comment:
I don't think this is `Required for all ship operations` anymore

Reply:
Do we even want to support the `ship init replicated.app/<name> --customer-id=A --installation-id=B` form rather than the `replicated.app/<name>?customer_id=A&installation_id=B` form?